client/src/components/FilterPreview.js

```diff
@@ -1,23 +1,38 @@
-import React, { Component } from 'react';
-import '../css/FilterPreview.css';
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Card } from 'antd';
 import { Link } from 'react-router-dom';
 
-export default class FilterPreview extends Component<Props, State> {
-  constructor(props) {
-    super(props);
-    this.state = {};
-  }
+const { Meta } = Card;
 
-  render() {
-    const { id } = this.props;
+function FilterPreview(props) {
+  const { availableLanguages, cost, id, name } = props;
 
-    return (
-      <Link to={`resource/${id}`}>
-        <div className="filter-preview">
-          <div className="filter-preview-top" />
-          <div className="filter-preview-bottom">{this.props.resourceName}</div>
-        </div>
-      </Link>
-    );
+  const description = [];
+  let languages = '';
+  availableLanguages.forEach(language => {
+    languages += `${language}, `;
+  });
+  if (languages !== '') {
+    languages = languages.slice(0, languages.length - 2);
   }
+  description.push(<div style={{ color: '#431C72' }}>{cost}</div>);
+  description.push(<div style={{ color: 'black' }}>{languages}</div>);
+
+  return (
+    <Link to={`resource/${id}`}>
+      <Card cover={<img alt="ymca" src="/logo512.png" />}>
```

Comment:
I think that this is failing (you can see at https://ymca-git-filter-ui-improvements.hack4impact1.now.sh/filter) because this file actually isn't deployed. To fix this, add the following in the [`now.json`](https://github.com/hack4impact-uiuc/ymca/blob/master/now.json) (probably under where we map `favicon.ico`)

```
{ "src": "/logo512.png", "dest": "/client/logo512.png" },
```